packages/amplify-category-storage/src/provider-utils/awscloudformation/import/import-s3.ts

```diff
@@ -464,8 +466,8 @@ const ensureHeadlessParameters = (headlessParams: ImportS3HeadlessParameters): S
   }
 
   const envSpecificParameters: S3EnvSpecificResourceParameters = {
-    bucketName: headlessParams.bucketName,
-    region: headlessParams.region,
+    bucketName: headlessParams.bucketName!,
```

Comment:
If this is truly _always_ non-null, we should be able to safely handle the non-null case safely. This could be wrapped in an if statement that throws a `TypeError` if either of those values are null.

Reply:
This was a leftover from an earlier revision I meant to remove, good catch!